element_facemap/facial_behavior_estimation.py

```diff
@@ -394,12 +502,16 @@ def make(self, key):
 
 
 def get_loader_result(key, table):
-    """
-    Retrieve the results from the facemap loader
-        :param key: the `key` to one entry of FacemapTask
-        :param table: the class defining the table to retrieve
-         the loaded results from (e.g. FacemapTask)
-        :return: output dictionary in the _proc.npy and the creation date time
+    """Retrieve the facemap analysis results.
+
+    Args:
+        key (dict): A primary key to an entry in FacemapTask.
+        table (dj.Computed): The class defining the table to retrieve the loaded results
+            from (e.g. FacemapTask)
```

Comment:
```suggestion
        table (dj.Table): The class defining the table to retrieve the loaded results
            from (e.g. FacemapTask)
```

Reply:
Thanks for your review @kabilar - I've made this adjustment in my most recent commit